plugins/Installation/tests/System/APITest.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\Installation\tests\System;
+
+use Piwik\Config;
+use Piwik\Tests\Framework\Constraint\HttpResponseText;
+use Piwik\Tests\Framework\Fixture;
+use Piwik\Tests\Framework\TestCase\SystemTestCase;
+
+/**
+ * @group Installation
+ * @group APITest
+ * @group Plugins
+ */
+class APITest extends SystemTestCase
+{
+    /**
+     * @var Fixture
+     */
+    public static $fixture = null; // initialized below class definition
+
+    public static function setUpBeforeClass()
+    {
+        parent::setUpBeforeClass();
+
+        $testingEnvironment = new \Piwik\Tests\Framework\TestingEnvironmentVariables();
+        $testingEnvironment->configFileLocal = PIWIK_INCLUDE_PATH . '/plugins/Installation/tests/resources/config.ini.php';
+        $testingEnvironment->save();
+    }
```

Comment:
NOTE: It should also be possible to set `configFileLocal` by overriding the `provideContainerConfig()`, eg:

```
public function provideContainerConfig()
{
    return array(
        'test.vars.configFileLocal' => PIWIK_INCLUDE_PATH . '/plugins/...';
    );
}
```

This isn't documented, but is intended to eventually replace direct `TestingEnvironmentVariables()` and `TestingEnvironmentVariables::save()` use. It's not really important, but if you'd like, you can use `provideContainerConfig()` instead here.